Makefile

```diff
@@ -239,6 +239,15 @@ ct-mnesia-rocksdb: SYSCONFIG=config/test-lima.config
 ct-mnesia-rocksdb: AETERNITY_TESTCONFIG_DB_BACKEND=rocksdb
 ct-mnesia-rocksdb: internal-ct
 
+stratum-client-internal-build:
+	@(cd ./_build/$(KIND)/ && \
+	  if [ ! -d aestratum_client ]; then \
+	      git clone --branch v0.1 --depth 1 https://github.com/aeternity/aestratum_client; \
```

Comment:
If https://github.com/aeternity/aestratum_client/pull/2 accepted this needs change.

Reply:
The change is now pushed to dev-stratum, pulling v0.1.1 version of client.